NOTES.md

```diff
@@ -0,0 +1,36 @@
+## Notes API
+
```

Comment:
Great documentation - very thorough. I really like the example of the basic note format along with the descriptions of your endpoints.